.travis.yml

```diff
@@ -11,12 +11,15 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
 matrix:
   include:
     - php: 5.3
       env: deps=low
+  allow_failures:
+    - php: 7.0
```

Comment:
Tests don't fail on PHP 7 right now. What about removing this to catch future regressions?